apps/server/src/infra/etherpadClient/loggable/etherpad-cookies-configuration-missing.loggable.ts

```diff
@@ -0,0 +1,30 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+
+export enum MissingCookie {
+	cookieReleaseThreshold,
+	cookieExpiration,
+}
```

Comment:
Please move this enum to a separate file (I think it should be also placed in an `interface` dir) also please use "CONSTANT_CASE" convention for the enum keys.

Reply:
done